plugins/woocommerce/client/legacy/js/admin/meta-boxes-product-variation.js

```diff
@@ -1317,41 +1327,48 @@ jQuery( function ( $ ) {
 					);
 
 					if ( null === data ) {
-						return;
+						cancel = true;
 					}
 					break;
 			}
 
-			if ( 'delete_all' === do_variation_action && data.allowed ) {
-				$( '#variable_product_options' )
-					.find( '.variation-needs-update' )
-					.removeClass( 'variation-needs-update' );
-				$( '.generate_variations' ).text( 'Generate variations' );
+			if( cancel ) {
+				document.getElementById( 'field_to_edit' ).selectedIndex = 0;
```

Comment:
While there's technically nothing wrong with this, all the code in this file is using jQuery to fetch the `#field_to_edit` field. For consistency, maybe we could do this using jQuery too?

Maybe something like `$( '#field_to_edit' ).val( 'bulk_actions' );` 

Obviously not a blocker, but this part does stand out given all the code around it. Same below.

Reply:
I always try to [KISS](https://en.wikipedia.org/wiki/KISS_principle) and so I use built-in language/framework funcionality instead of external libraries whenever I can. But it's true that this seems inconsistent. I have chaned it to use JQuery in [`432499d` (#54399)](https://github.com/woocommerce/woocommerce/pull/54399/commits/432499d06c88b66b909bfa8b2ac9c511e42b2011).